src/executor/Executor.cc

```diff
@@ -68,6 +70,13 @@ void Executor::executeChild() {
         listener->onPostForkChild();
     }
 
+    if (childProgramWorkingDir_ != "") {
```

Comment:
How about `.empty()` ? It'd be more consistent.

Reply:
Ok, I changed it.